game/src/lib.rs

```diff
@@ -415,6 +419,8 @@ fn finish_app_setup(
     } else if setup.ungap {
         let layers = ungap::Layers::new(ctx, app);
         vec![ungap::ExploreMap::new_state(ctx, app, layers)]
+    } else if setup.ltn {
```

Comment:
I've wedged the tool into the game crate for the moment. If we decide to develop it further, it should definitely be split into a proper crate of its own.